views/default/elgg/ajax.js

```diff
@@ -0,0 +1,199 @@
+define(function (require) {
+	var $ = require('jquery');
+	var elgg = require('elgg');
+	var spinner = require('elgg/spinner');
+
+	var spinner_enabled = true;
```

Comment:
Could we make it a global config option? Then spinner can only be required if it's enabled.